xformers/csrc/attention/hip_fmha/instances/\

```diff
@@ -0,0 +1,12 @@
+/*
```

Comment:
did you mean to commit this file?

Reply:
It looks like the template instance is needed, but the filename should be different

Reply:
Actually, it's a duplicate; fixed